pythran/backend.py

```diff
@@ -955,7 +955,10 @@ def visit_Call(self, node):
             return "{}({})".format(func, ", ".join(args))
 
     def visit_Constant(self, node):
-        if node.value is None:
+        if node in self.immediates and isinstance(node.value, bool):
+            ret = "std::integral_constant<%s, %s>{}" % (
+                PYTYPE_TO_CTYPE_TABLE[type(node.value)], node.value)
```

Comment:
`node.value` will print `True` instead of `true`. You should use `str(node.value).lower()` instead